src/app/containers/ATIAnalytics/index.test.jsx

```diff
@@ -123,7 +123,7 @@ describe('ATI Analytics Container', () => {
 
       expect(mockAmp.mock.calls[0][0]).toMatchInlineSnapshot(`
         Object {
-          "pageviewParams": "s=598286&s2=64&p=news.page&r=%24%7BscreenWidth%7Dx%24%7BscreenHeight%7Dx%24%7BscreenColorDepth%7D&re=%24%7BavailableScreenWidth%7Dx%24%7BavailableScreenHeight%7D&hl=00-00-00&lng=%24%7BbrowserLanguage%7D&x2=[amp]&x3=[news]&x5=[%24%7BsourceUrl%7D]&x6=[%24%7BdocumentReferrer%7D]&x7=[index-home]&x8=[simorgh]&x11=[1970-01-01T00%3A00%3A00.000Z]&x12=[1970-01-01T00%3A00%3A00.000Z]&ref=\${documentReferrer}",
+          "pageviewParams": "s=598286&s2=64&p=news.page&r=%24%7BscreenWidth%7Dx%24%7BscreenHeight%7Dx%24%7BscreenColorDepth%7D&re=%24%7BavailableScreenWidth%7Dx%24%7BavailableScreenHeight%7D&hl=00-00-00&lng=%24%7BbrowserLanguage%7D&x2=[amp]&x3=[news]&x5=[%24%7BsourceUrl%7D]&x6=[%24%7BdocumentReferrer%7D]&x7=[index-home]&x8=[simorgh]&x11=[1970-01-01T00%3A00%3A00.000Z]&x12=[1970-01-01T00%3A00%3A00.000Z]&ref=%24%7BdocumentReferrer%7D",
```

Comment:
do we need encoding for amp params as well ?

Reply:
This is actually the revert of the logic, I accidentally pushed this to latest without a review 🙈